src/Microsoft.Identity.Web.DownstreamApi/DownstreamApi.cs

```diff
@@ -1,234 +1,235 @@
-            DownstreamApiOptions effectiveOptions = MergeOptions(serviceName, downstreamApiOptionsOverride);
-            HttpResponseMessage response = await CallApiInternalAsync(serviceName, effectiveOptions, false,
-                                                                          null, user, cancellationToken).ConfigureAwait(false);
-            return await DeserializeOutput<TOutput>(response, effectiveOptions).ConfigureAwait(false);
-        }
-
-        /// <summary>
-        /// Merge the options from configuration and override from caller.
-        /// </summary>
-        /// <param name="optionsInstanceName">Named configuration.</param>
-        /// <param name="calledApiOptionsOverride">Delegate to override the configuration.</param>
-        internal /* for tests */ DownstreamApiOptions MergeOptions(
-            string? optionsInstanceName,
-            Action<DownstreamApiOptions>? calledApiOptionsOverride)
-        {
-            // Gets the options from configuration (or default value)
-            DownstreamApiOptions options;
-            if (optionsInstanceName != null)
-            {
-                options = _namedDownstreamApiOptions.Get(optionsInstanceName);
-            }
-            else
-            {
-                options = _namedDownstreamApiOptions.CurrentValue;
-            }
-
-            DownstreamApiOptions clonedOptions = new DownstreamApiOptions(options);
-            calledApiOptionsOverride?.Invoke(clonedOptions);
-            return clonedOptions;
-        }
-
-        /// <summary>
-        /// Merge the options from configuration and override from caller.
-        /// </summary>
-        /// <param name="optionsInstanceName">Named configuration.</param>
-        /// <param name="calledApiOptionsOverride">Delegate to override the configuration.</param>
-        /// <param name="httpMethod">Http method overriding the configuration options.</param>
-        internal /* for tests */ DownstreamApiOptions MergeOptions(
-            string? optionsInstanceName,
-            Action<DownstreamApiOptionsReadOnlyHttpMethod>? calledApiOptionsOverride, HttpMethod httpMethod)
-        {
-            // Gets the options from configuration (or default value)
-            DownstreamApiOptions options;
-            if (optionsInstanceName != null)
-            {
-                options = _namedDownstreamApiOptions.Get(optionsInstanceName);
-            }
-            else
-            {
-                options = _namedDownstreamApiOptions.CurrentValue;
-            }
-
-            DownstreamApiOptionsReadOnlyHttpMethod clonedOptions = new DownstreamApiOptionsReadOnlyHttpMethod(options, httpMethod.ToString());
-            calledApiOptionsOverride?.Invoke(clonedOptions);
-            return clonedOptions;
-        }
-
-        internal static HttpContent? SerializeInput<TInput>(TInput input, DownstreamApiOptions effectiveOptions)
-        {
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
can you remove the format changes so it's easier to review this file?

Reply:
TLDR; Use the "hide whitespace changes" PR review option
![image](https://github.com/user-attachments/assets/ad73f11b-464b-4860-9df7-aa87a4b27351)

I am using Visual Studio defaults. Should I use smth else?
In my experience this happens when you make edits to files via GitHub PR system (e.g. inject suggestions). GH uses Linux style line endings and VS windows style. This results in a mix and VS complains: 

![image](https://github.com/user-attachments/assets/92c34669-7010-48ab-83ba-4c6ac22244e8)


For example you can try opening the file `DownstreamApiExtensions.cs` and you will see VS complaining. 

In any case, line endings are invisible chars, so I have no idea how to undo this.